### DISCUSSION TITLE
홈 화면 사이드바와 글 리스트 사이 간격을 넓고 싶습니다.

### DISCUSSION THREAD

**Discussion by kgw7401** [pronist/hELLO#198] [Q&A] [answered]
홈 화면 사이드바와 글 리스트 사이 간격이 너무 딱 붙어있는거 같아 이 사이 간격을 넓히고 싶습니다. 어디 부분을 건드리면 좋을까요?

**Comment by pronist** [comment, marked as answer]
@kgw7401 안녕하세요.

사이드바가 화면에서 사라지는 가로 사이즈는 현재 `1400px` 로 설정되어 있습니다. 클라이언트 환경이 `1400px` 에 근접한다면 사이드바가 노출이 된 상태로 홈화면이 보여지므로 간격이 작아보일 수 있습니다. 이 스킨은 조금 더 넓은 FHD 를 기준 사이즈로 제작되었기 때문입니다.

https://github.com/tistory-projects/tistory-skin-hELLO/blob/d89d0075d1d3d2e019d17f33af28f92053cb4f46/tailwind.config.js#L12

홈 화면과 글목록은 `1100px` 고정입니다.

https://github.com/tistory-projects/tistory-skin-hELLO/blob/d89d0075d1d3d2e019d17f33af28f92053cb4f46/tailwind.config.js#L34

이 설정은 원본 소스코드에서 tailwind.config.js 에서 찾아볼 수 있으며 스킨 배포파일인 skin.html 이나 style.css 에서 찾아보는 것은 상당히 불편할 수 있습니다.
